xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamNavigator.cpp

```diff
@@ -1026,60 +1026,60 @@ void CDVDInputStreamNavigator::SetAudioStreamName(AudioStreamInfo &info, const a
   switch(audio_attributes.audio_format)
   {
   case DVD_AUDIO_FORMAT_AC3:
-    info.name += " AC3";
+    info.codecDesc = "AC3";
     info.codecName = "ac3";
     break;
   case DVD_AUDIO_FORMAT_UNKNOWN_1:
-    info.name += " UNKNOWN #1";
+    info.codecDesc = g_localizeStrings.Get(13205); // UNKNOWN
+    CLog::LogF(LOGINFO, "unknown dvd audio codec DVD_AUDIO_FORMAT_UNKNOWN_1");
     break;
   case DVD_AUDIO_FORMAT_MPEG:
-    info.name += " MPEG AUDIO";
+    info.codecDesc = g_localizeStrings.Get(37003); // MPEG AUDIO
     info.codecName = "mp1";
     break;
   case DVD_AUDIO_FORMAT_MPEG2_EXT:
-    info.name += " MP2 Ext.";
+    info.codecDesc = "MP2 Ext.";
```

Comment:
here can be also possible dont use translations just use full length name
"MPEG-1" for DVD_AUDIO_FORMAT_MPEG
"MPEG-2" for DVD_AUDIO_FORMAT_MPEG2_EXT